controller/src/test/java/io/pravega/controller/server/eventProcessor/requesthandlers/RequestProcessorWithInMemoryStore.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server.eventProcessor.requesthandlers;
+
+import io.pravega.controller.store.stream.StreamMetadataStore;
+import io.pravega.controller.store.stream.StreamStoreFactory;
+import org.junit.After;
+import org.junit.Before;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+
+public class RequestProcessorWithInMemoryStore extends RequestProcessorTest {
```

Comment:
Please have the base class extend from `ThreadPooledTestSuite` (and set the number of executors you want). That takes care of  managing the executor, including the proper way of shutting it down (what you do in `tearDown` is not the right way, BTW).